style.css

```diff
@@ -0,0 +1,39 @@
+body{
+    background-color: lavender;
+   
+}
+
+header{
```

Comment:
We can use the `align-items` property here to align the text in the middle via y-axis